views/show.blade.php

```diff
@@ -84,12 +84,12 @@
 @stop
 
 @section('css')
-{!! Asset::styles('logviewer') !!}
+    {!! HTML::style(asset('styles/logviewer.css') !!}
```

Comment:
In terms of the asset function, I don't think I'm that bothered tbh. Anyone running a custom framework will need just just define that function.